README.md

```diff
@@ -2,11 +2,12 @@
 
 # Hello Microverse
 
-> I create my first portafolio Proyect with Microverse
+> I create my first Portfolio the set up and mobile first
+
 
 ## Author
 
-👤 **Author1**
+👤 **Autor1**
```

Comment:
Kindly, in this place you should put your name

Reply:
i alredy change the name